app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -65,5 +55,16 @@ def textual_product_name
       name = @record.operating_system.try(:product_name)
       {:label => "Product Name", :value => name} if name
     end
+
+    def textual_guest_applications
```

Comment:
I moved textual_guest_applications to container_summary_helper and made it use textual_link.
My only problem is that I can't make it work without sending the :link. @matthewd @h-kataria can you help?